DTS-STN/canadian-dental-care-plan
# AB#2659 -- `/auth/logout` prepwork

### Description

Adding some envars that will eventually be used in the `/auth/logout` route. This is an incremental PR that exists solely to keep future PRs smaller.

### Related Azure Boards Work Items

- [AB#2659](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2659)
- [AB#2915](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2915)

### Screenshots

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/f6605c4b-c90a-424d-afb4-7cbb945bb63f)

### Checklist


### Test Instructions

Copy new environment variable to your `.env` file; verify that the application runs.

<details>
  <summary>Warning: spoiler</summary>  
  <h3>IT DOES!!!</h3>
</details>
